src/main/java/com/booster/server/VocabularyEntryService.java

```diff
@@ -35,4 +39,12 @@ private VocabularyEntryDto vocabularyEntryDto(VocabularyEntryEntity entity) {
                 .setDescription(entity.getDescription());
     }
 
+    public void updateLastSeenAt(UpdateLastSeenAtInput input) {
+        List<VocabularyEntryEntity> updatedVocabularyEntries = vocabularyEntryRepository.findAllById(input.getIds())
+                .stream()
+                .peek(entry -> entry.setLastSeenAt(input.getLocalDateTime()))
+                .toList();
+        vocabularyEntryRepository.saveAll(updatedVocabularyEntries);
+    }
```

Comment:
I am wondering if we need a `Stream` roundtrip here at all.

What if we do smth like this:
```java
List<VocabularyEntryEntity> entries = vocabularyEntryRepository.findAllById(input.getIds());
entries.forEach(e -> e.setLastSeenAt(input.getLocalDateTime()));
vocabularyEntryRepository.saveAll(entries);
```
This way we make the operation more performant.